src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/Android/PopupRenderer.android.cs

```diff
@@ -124,7 +128,18 @@ void SetEvents()
 
 		void SetColor()
 		{
-			Window.SetBackgroundDrawable(new ColorDrawable(Element.Color.ToAndroid()));
+			if (Element.Color == Color.Default)
+			{
+				var color = Build.VERSION.SdkInt >= BuildVersionCodes.M ?
+					Context.Resources.GetColor(AColorRes.BackgroundLight, Context.Theme) :
+					new AColor(ContextCompat.GetColor(Context, AColorRes.BackgroundLight));
+
+				Window.SetBackgroundDrawable(new ColorDrawable(color));
+			}
+			else
+			{
+				Window.SetBackgroundDrawable(new ColorDrawable(Element.Color.ToAndroid()));
+			}
```

Comment:
This logic can be moved to a separate method for improving readability :)

```csharp
AColor GetWindowColor() {
	if (Element.Color != Color.Default)
		return Element.Color.ToAndroid();
	
	return Build.VERSION.SdkInt >= BuildVersionCodes.M 
		? Context.Resources.GetColor(AColorRes.BackgroundLight, Context.Theme) 
		: new AColor(ContextCompat.GetColor(Context, AColorRes.BackgroundLight));
}
```

And then just:

```csharp
Window.SetBackgroundDrawable(new ColorDrawable(GetWindowColor()));
```

Reply:
A pity there ain’t a public Xamarin forms function we can use here..
Made the changes you requested.